addons/mail/static/src/owl/components/chatter_topbar/chatter_topbar.scss

```diff
@@ -8,15 +8,38 @@
     justify-content: space-between;
 }
 
+.o_ChatterTopbar_buttonAttachmentsCountLoader {
+    margin-left: 2px;
+}
+
 .o_ChatterTopbar_buttonCount {
-    padding-left: 0.25rem !important;
+    padding-left: 0.25rem;
 }
 
+.o_ChatterTopbar_buttonSendMessage {
+    padding: 0 16px;
+    border-left: none;
+}
+
+.o_ChatterTopbar_leftSection {
+    display: flex;
+}
 
 // ------------------------------------------------------------------
 // Style
 // ------------------------------------------------------------------
 
+.o_ChatterTopbar {
+
+    .o-active {
+        color: $o-brand-odoo;
+        background-color: lighten(gray('300'), 7%);
+        border-color: $gray-300;
```

Comment:
isn't it equivalent to `gray('300')`?

Reply:
Possibly but isn't it better to use global style variables ?

Reply:
Sure, prefer variable than hard-code a colour hex-value that is used in many places, usually because variables are more meaningful than hex-values.

However, in this case, this variable seems to align behaviour with `gray('300')`, which is the standard way of having a certain shade of gray.

Reply:
I changed as through all mail module `gray('300')` is used but `$gray-300$` is never used but there are multiple definition of `$gray-300$` so I'm not sure that it's "that" simple (I don't have enough experience with Odoo to determine that)